### PR TITLE
Fix token reset while running tests

### LINKED ISSUE #72: Auth Token value gets reset
The token which is placed at `~/.evalai/token.json` gets reset whenever a user runs test cases on his local machine.

**Steps to reproduce:**
1. Add a valid token at `~/.evalai/token.json`.
2. Run the test cases using `py.test`.
3. Check the value of token at `~/.evalai/token.json`

The token `value at step 1 != value at step 3`.

### PR DESCRIPTION
**Pull request by isht3**
Fixes #72

### CODE REVIEW

File: tests/test_auth.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import json
 import os
 import responses
-import shutil
+
+import os.path as osp
```
Comment (RishabhJain2018) [inline tests/test_auth.py]: Please remove this line as we already have imported `os module`

File: tests/test_auth.py [hunk 2]
```diff
@@ -19,9 +21,16 @@
 
 class TestGetUserAuthToken(BaseTestClass):
 
+    token_file = osp.join(AUTH_TOKEN_DIR, AUTH_TOKEN_FILE_NAME)
```
Comment (RishabhJain2018) [inline tests/test_auth.py]: Please change `osp` to `os.path`

File: tests/base.py [hunk 3]
```diff
@@ -14,9 +14,10 @@ class BaseTestClass:
 
     @classmethod
     def setup_class(cls):
-        if not os.path.exists(AUTH_TOKEN_DIR):
+        token_file = osp.join(AUTH_TOKEN_DIR, AUTH_TOKEN_FILE_NAME)
```
Comment (RishabhJain2018) [inline tests/base.py]: Please change osp to `os.path`. Also remove line 6 from this file.

**Comment by isht3** [comment]
@RishabhJain2018 Changes made!

File: tests/base.py [hunk 4]
```diff
@@ -14,9 +12,10 @@ class BaseTestClass:
 
     @classmethod
     def setup_class(cls):
-        if not os.path.exists(AUTH_TOKEN_DIR):
+        token_file = os.path.join(AUTH_TOKEN_DIR, AUTH_TOKEN_FILE_NAME)
+        if not os.path.exists(token_file):
             os.makedirs(AUTH_TOKEN_DIR)
-            with open(osp.join(AUTH_TOKEN_DIR, AUTH_TOKEN_FILE_NAME), 'w') as fw:
+            with open(token_file, 'w') as fw:
```
Comment (varunagrawal) [inline tests/base.py]: Is the token saved as plaintext? Should it be encrypted in any way?

Comment (RishabhJain2018) [reply tests/base.py]: yes, it is! But yes I think we should encrypt it. @isht3 Can you please add that change in a new PR.

Comment (deshraj) [reply tests/base.py]: We don't need to encrypt the token since it's already encrypted by the django-rest-auth package.

**Pull request merged**